src/naive/time/arbitrary.rs

```diff
@@ -0,0 +1,44 @@
+#![cfg_attr(docsrs, doc(cfg(feature = "arbitrary")))]
+
+use super::NaiveTime;
+use arbitrary::{Arbitrary, Unstructured};
+
+impl Arbitrary<'_> for NaiveTime {
+    fn arbitrary(u: &mut Unstructured) -> arbitrary::Result<NaiveTime> {
+        let secs = u.int_in_range(0..=86_399)?;
+        let nano = u.int_in_range(0..=1_999_999_999)?;
+        let time = NaiveTime::from_num_seconds_from_midnight_opt(secs, nano)
```

Comment:
What's the rationale for panicking here vs returning an error?

Reply:
My primary motivation was this comment https://github.com/chronotope/chrono/pull/559#discussion_r996506593, which you thumbed up.
I am OK with both approaches, both have their pros and cons. With this one (unwrap/expect) we'll detect the problem faster if somethings gets broken. (Arbitrary is implement that way, that if something returns an error, it retries to generate data until it succeeds).